.github/workflows/k8s-validation.yaml

```
@@ -15,8 +21,9 @@ jobs:
     - name: Generate manifests
       run: |
         mkdir manifests
-        kubectl kustomize k8s/staging > manifests/staging.yaml
-        kubectl kustomize k8s/production > manifests/production.yaml
+        for env in ${${{ inputs.environments }}//,/ } do
```

Ich glaube hier fehlt ein `;` vor dem `do` und außerdem würde ich die Liste glaub ich fast eher "space-separated" machen, weil das dann (etwas) einfacher wird:
```suggestion
        for env in ${{ inputs.environments }}; do
```